### PR TITLE
Fix amp-crypto-polyfill module error

### LINKED ISSUE #18851: Q$$module$third_party$closure_library$sha384_generated is not defined at Q
Error: Q$$module$third_party$closure_library$sha384_generated is not defined at Q (https://raw.githubusercontent.com/ampproject/amphtml/1810191936340/third_party/closure-library/sha384-generated.js:11:555) at https://raw.githubusercontent.com/ampproject/amphtml/1810191936340/src/custom-element.js:1387:8 at https://raw.githubusercontent.com/ampproject/amphtml/1810191936340/src/service/viewport/viewport-impl.js:1108:30

### PR DESCRIPTION
**Pull request by choumx**
Fixes #18851. 

- Needed to change output wrapper to IIFE to fix unit tests.
- We really should add integration tests for these.

### CODE REVIEW

File: third_party/closure-library/compile.sh [hunk 1]
```diff
@@ -28,9 +28,11 @@ java -jar $1 \
     --js "$2/goog/reflect/reflect.js" \
     --js "$2/goog/string/string.js" \
     --js_output_file "sha384-generated.js" \
-    --output_wrapper "/* Generated from closure library commit $GIT_COMMIT
-*/%output%;exports.base64=function base64(input) { return ampBase64(input) };
```
Comment (erwinmombay) [inline third_party/closure-library/compile.sh]: do we just not use base64 anywhere?

Comment (choumx) [reply third_party/closure-library/compile.sh]: Yea, we don't even export `ampBase64` in sha384.js.

File: third_party/closure-library/compile.sh [hunk 2]
```diff
@@ -28,9 +28,11 @@ java -jar $1 \
...
-exports.sha384=function sha384(input) { return ampSha384Digest(input) };" \
+    --output_wrapper "/* Generated from closure library commit $GIT_COMMIT */
+export const sha384 = (function(){
+%output%;
+return ampSha384Digest;
+}).call(window);" \
```
Comment (jridgewell) [inline third_party/closure-library/compile.sh]: Nit: `self`.

**Comment by cramforce** [comment]
Approved

**Pull request merged**